charts/rasa-x/templates/db-migration-service.yaml

```diff
@@ -0,0 +1,21 @@
+{{ if eq "true" (include "db-migration-service.requiredVersion" .) }}
+{{ if .Values.separateDBMigrationService }}
+apiVersion: "v1"
+kind: "Service"
+metadata:
+  name: {{ include "rasa-x.fullname" . }}-db-migration-service-headless
+  labels:
+    {{- include "rasa-x.labels" . | nindent 4 }}
+    app.kubernetes.io/component: app
+spec:
+  clusterIP: "None"
+  ports:
+  - port: {{ .Values.dbMigrationService.port }}
+    targetPort: {{ .Values.dbMigrationService.port }}
+    protocol: "TCP"
+    name: "http"
+  selector:
+    {{- include "rasa-x.selectorLabels" . | nindent 4 }}
+    app.kubernetes.io/component: db-migration-service
```

Comment:
haha, the `service` confused me very much here 😄

Reply:
That's true 😄